src/Ledger/Conway/Specification/Epoch.lagda.md

```diff
@@ -454,11 +463,34 @@ opaque
         where
           govSt' : ℙ (GovActionID × RwdAddr)
           govSt' = mapˢ (map₂ returnAddr) (fromList govSt)
+```
+
+```agda
+  calculatePoolDelegatedStakeForVoting
+    : Snapshot
+    → UTxOState
+    → GovState
+    → GState
+    → DState
+    → KeyHash ⇀ Coin
+  calculatePoolDelegatedStakeForVoting ss utxoSt govSt gState dState
+    = calculatePoolDelegatedStake ss ∪⁺ stakeFromDeposits
+    where
+      open UTxOState utxoSt
+
+      stakeFromGADeposits : Stake
+      stakeFromGADeposits = aggregateBy
+        (mapˢ (λ (gaid , addr) → (gaid , addr) , stake addr) govSt')
+        (mapFromPartialFun (λ (gaid , _) → lookupᵐ? deposits (GovActionDeposit gaid)) govSt')
+        where
+          govSt' : ℙ (GovActionID × RwdAddr)
+          govSt' = mapˢ (map₂ returnAddr) (fromList govSt)
 
-      -- stake from rewards
-      stakeFromRewards : Stake
-      stakeFromRewards = RewardsOf dState
```

Comment:
This is duplicated since the function `stakeDistr` in `Rewards` already takes into account rewards